src/lib/djerba/plugins/genomic_landscape/plugin.py

```diff
@@ -29,7 +29,7 @@ class main(plugin_base):
     TEMPLATE_NAME = 'genomic_landscape_template.html'
 
     # For MSI file
-    MSI_RESULTS_SUFFIX = '.filter.deduped.realigned.recalibrated.msi.booted'
+    MSI_RESULTS_SUFFIX = '.filter.deduped.recalibrated.msi.booted'
```

Comment:
Can we use `recalibrated.msi.booted` as the suffix? That will allow both variations, and looking at provenance, it seems to be sufficient to uniquely identify the file.